index.html

```diff
@@ -14,6 +14,7 @@
     <h1 class="box">The Game of Life</h1>
     <p>Click on the grid to choose the game's initial configuration</p>
     <p>Click play and watch the game evolve</p>
+    <p>Click on the grid to choose the game's initial configuration OR click on the randomize button</p>
```

Comment:
I believe this line is meant to replace and enrich line 15

Reply:
Will do.